app/styles/components/contributor-list.scss

```diff
@@ -1,4 +1,4 @@
-.meeting-keynote-speakers-container {
+.contributor-list-container {
 
   .speaker-pic-thumbnail {
```

Comment:
do we want to use speaker or contributor?

Reply:
contributor - good point again!